cmd/infracost/output.go

```diff
@@ -31,15 +31,15 @@ func outputCmd(ctx *config.RunContext) *cobra.Command {
 
   Create HTML report from multiple Infracost JSON files:
 
-      infracost output --format html --path "out*.json" > output.html
+      infracost output --format html --path "out*.json" --out-file output.html (glob needs quotes)
 
   Merge multiple Infracost JSON files:
 
-      infracost output --format json --path "out*.json"
+      infracost output --format json --path "out*.json" (glob needs quotes)
```

Comment:
do we need this and the next mention or is the line 34 enough?

Reply:
doesn't hurt